### ISSUE TITLE
Make tests less dependent on environment

### ISSUE THREAD

**Issue by maxnikulin** [danny0838/PyWebScrapBook#20]
I have spent some time trying to avoid test failures on linux. Have a look at https://github.com/maxnikulin/PyWebScrapBook/tree/test-conditions

1. A test calling `mklink` is not marked as windows-only
2. I do not see any reason to compare file access time. It is not a backup or archiving application and there are some tricky issues with preventing of updating of access time.

    ```
    FAIL: test_symlink_deep (test_app_actions.TestCopy)
    Copy symlink entities with referenced directory content.
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/usr/lib/python3.8/unittest/mock.py", line 1348, in patched
        return func(*newargs, **newkeywargs)
      File "/home/scrapbook-src/PyWebScrapBook-dev/test/test_app_actions.py", line 4506, in test_symlink_deep
        self.assert_file_equal(
      File "/home/scrapbook-src/PyWebScrapBook-dev/test/test_app_actions.py", line 173, in assert_file_equal
        self.assertAlmostEqual(stat0[i], stati[i], delta=2)
    AssertionError: 1600322648.18454 != 31536000.0 within 2 delta (1568786648.18454 difference)
    ```

3. Date and time fields are stored in Zip files, not timestamps since Zip is unaware of time zones. As a result success of zip-related tests depends on running in particular time zone due to namely timestamps are hardcoded. I am tempting to use pytest and their fixtures as free standing functions for creation of a test zip file with info that could be verified. It is much more convenient than JUnit setUp/tearDown functions since could be added to any test independent of particular module. Are you against pytest?

    For a while I have modified a couple of tests. Unfortunately more functions require similar changes.

**Comment by danny0838** [comment]
Implemented in 0.21.0

**Comment by maxnikulin** [comment]
Sorry, more tests need similar modification, so #21
